torch/__init__.pyi.in

```diff
@@ -122,6 +122,7 @@ class Tensor:
     ${tensor_method_hints}
 
     # Manually defined methods from torch/tensor.py
+    def __init__(self, arg: Any = None): ...
```

Comment:
There's another way one can invoke the constructor. `torch.Tensor(1, 2, 3)` creates a tensor with uninitialized values of shape [1, 2, 3]. But also please see my comment in https://github.com/pytorch/pytorch/issues/37722#issuecomment-624069895